RELEASE-NOTES.md

```diff
@@ -16,6 +16,7 @@
 - `pm.LKJCholeskyCov` now automatically computes and returns the unpacked Cholesky decomposition, the correlations and the standard deviations of the covariance matrix (see [#3881](https://github.com/pymc-devs/pymc3/pull/3881)).
 
 ### Maintenance
+- AR1 model correctly calculates likelihood of the first observation of a sample, see [#3899](https://github.com/pymc-devs/pymc3/pull/3899)
```

Comment:
- Can we be more specific? Maybe "The AR1 logp now uses the precision of the whole AR1 process instead of just the innovation precision as before (see ...)"
- Can you put this at the end of the bullet points (the order is chronological)?